assets/queries/openAPI/parameter_object_incorrect_ref/query.rego

```diff
@@ -5,29 +5,17 @@ import data.generic.openapi as openapi_lib
 CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) != "undefined"
-	params := doc.paths[name].parameters[n]
-	not startswith(params["$ref"], "#components/parameters/")
 
-	result := {
-		"documentId": doc.id,
-		"searchKey": sprintf("openapi.paths.{{%s}}.parameters", [name]),
-		"issueType": "IncorrectValue",
-		"keyExpectedValue": "Parameter Object ref points to '#components/parameters'",
-		"keyActualValue": "Parameter Object ref doesn't point to '#components/parameters'",
-	}
-}
+	[path, value] := walk(doc)
 
-CxPolicy[result] {
-	doc := input.document[i]
-	openapi_lib.check_openapi(doc) != "undefined"
-	params := doc.paths[name][oper].parameters[n]
-	not startswith(params["$ref"], "#components/parameters/")
+	ref := value.parameters[n]["$ref"]
```

Comment:
I think we should ignore `components.parameters`, since it can have an external reference...